src/library/scala/collection/Iterator.scala

```diff
@@ -405,6 +413,7 @@ trait Iterator[+A] extends TraversableOnce[A] {
    *  @note    Reuse: $consumesAndProducesIterator
    */
   def filter(p: A => Boolean): Iterator[A] = new AbstractIterator[A] {
+    // TODO 2.12 - Make a full-fledged FilterImpl that will reverse sense of p
```

Comment:
Maybe wouldn't it be bad idea to have JIRA issues created for such TODOs? Then you could add here an issue number. WDYT?